tests/gnrc_ipv6_ext/README.md

```diff
@@ -0,0 +1,43 @@
+# `gnrc_ipv6_ext` test
+
+This test sends a packet to itself with extension headers. This is based on gnrc_networking example.
+
+Enable debug output of `gnrc_ipv6.c` before run. When the test is run, it should show the following debug output:
+
+```
+ipv6: Received (src = fd01::1, dst = fd01::2, next header = 0, length = 40)
+ipv6: forward nh = 0 to other threads
+ipv6: unable to forward packet as no one is interested in it
+ipv6: handle extension header (nh = 0)
+ipv6: forward nh = 43 to other threads
+ipv6: unable to forward packet as no one is interested in it
+ipv6: handle extension header (nh = 43)
+ipv6: waiting for incoming message.
+ipv6: GNRC_NETAPI_MSG_TYPE_RCV received
+ipv6: Received (src = fd01::1, dst = fd01::3, next header = 0, length = 40)
+ipv6: forward nh = 0 to other threads
+ipv6: unable to forward packet as no one is interested in it
+ipv6: handle extension header (nh = 0)
+ipv6: forward nh = 43 to other threads
+ipv6: unable to forward packet as no one is interested in it
+ipv6: handle extension header (nh = 43)
+ipv6: waiting for incoming message.
+ipv6: GNRC_NETAPI_MSG_TYPE_RCV received
+ipv6: Received (src = fd01::1, dst = fd01::2, next header = 0, length = 40)
+ipv6: forward nh = 0 to other threads
+ipv6: unable to forward packet as no one is interested in it
+ipv6: handle extension header (nh = 0)
+ipv6: forward nh = 43 to other threads
+ipv6: unable to forward packet as no one is interested in it
+ipv6: handle extension header (nh = 43)
+ipv6: forward nh = 17 to other threads
+ipv6: Send receive command for 0x1060b8 to 5
+ipv6: waiting for incoming message.
+pkt->users: 0
```

Comment:
Can you write a pexpect script for that? With @kaspar030's recent addition of a testsuite in #4965 this should be a breeze.